src/types.ts

```diff
@@ -6,11 +6,16 @@ export interface PeaqIns extends INSGeneric {
   SIGN: 0x02;
 }
 
-export interface ResponseAddress {
-  pubkey: Buffer;
+export interface ResponseBase {
+  error_message: string;
+  return_code: number;
+}
```

Comment:
Same for this. It seems pretty generic, and I guess it could be defined on ledger-js package instead.